30413/HulpeMiruna/4.2/main.c

```diff
@@ -0,0 +1,150 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+typedef struct node
+{
+    int label;
+    struct node *left, *right;
+} NodeTree;
+
+
+void initial(NodeTree *T)
+{
+    T->left=NULL;
+    T->right=NULL;
+    T->label=0;
+}
+
+NodeTree *createBinTree( int direction, NodeTree *T )
+{
+    NodeTree *p;
+    int id;
+    if ( direction == 0 )
+        printf( "Root identifier  =" );
+    else if ( direction == 1 )
+        printf( "Left child of %d  =",T->label );
+    else
+        printf( "Right child of %d  =",T->label );
```

Comment:
please use braces for only one line of code as well. Also.. please use the auto format functionality in CodeBlocks